charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -33,24 +33,24 @@ spec:
       containers:
         - name: driver-registrar
           image: "{{ .Values.nodeplugin.registrar.image.repository }}:{{ .Values.nodeplugin.registrar.image.tag }}"
+          imagePullPolicy: {{ .Values.nodeplugin.registrar.image.pullPolicy }}
           args:
             - "--v=5"
             - "--csi-address=/csi/{{ .Values.socketFile }}"
             - "--kubelet-registration-path={{ .Values.socketDir }}/{{ .Values.socketFile }}"
           lifecycle:
             preStop:
               exec:
-                  command: [
-                    "/bin/sh", "-c",
-                    'rm -rf /registration/{{ .Values.driverName }}
+                command: [
+                  "/bin/sh", "-c",
+                  'rm -rf /registration/{{ .Values.driverName }}
```

Comment:
please check a space is present after driver name
this should be something like `rm  -rf /abc/a bcd/d`

Reply:
Done by removing all spaces from the string